desktop/libs/notebook/src/notebook/connectors/trino.py

```diff
@@ -289,6 +289,13 @@ def close_statement(self, notebook, snippet):
     return {'status': 0}
 
   def close_session(self, session):
+    catalogs = self._show_catalogs()
```

Comment:
Is this required? Or a rebase issue?

Reply:
It was just testing

Reply:
I will remove before merging the commit.